docs/semgrep-code/policies.md

```diff
@@ -78,7 +78,7 @@ Semgrep Code provides three rule modes:
 | -------   | ------ |
 | Monitor   | Rules in **Monitor mode** display findings only in: <ul><li>Semgrep Cloud Platform</li><li>User-defined notifications</li></ul>Set rules to this mode to evaluate their true positive rate and other criteria you may have. By keeping rules in Monitor, developers do not receive potentially noisy findings in their PRs or MRs.  |
 | Comment   | Rules in **Comment mode** display findings in:<ul><li>Developers' PRs or MRs</li><li>Semgrep Cloud Platform</li><li>User-defined notifications</li></ul>Set rules that have met your performance criteria to this mode when you are ready to display findings to developers.     |
-| Block     | Rules in **Block mode** prevent a PR or MR from merging if Semgrep Code detects a finding from these rules.<br />These rules display findings in:<ul><li>Developers' PRs or MRs</li><li>Semgrep Cloud Platform</li><li>User-defined notifications</li></ul>These are typically high-confidence, high-severity rules. |
+| Block     | Rules in **Block mode** cause the scan job to fail with an exit code of `1` if Semgrep Code detects a finding from these rules. You can use this result to enforce a hard block on the PR or MR. For example, GitHub users can enable branch protection and set the PR to fail if the Semgrep step fails. <br />These rules display findings in:<ul><li>Developers' PRs or MRs</li><li>Semgrep Cloud Platform</li><li>User-defined notifications</li></ul>These are typically high-confidence, high-severity rules. |
```

Comment:
I like "enforce a block on the PR or MR" for both of these phrases - "set up" above feels a bit weird because they may well already be set up, and "hard" feels a bit weird here because many blocks can be overridden by admins.

Otherwise love this though, will leave this up to you.

Reply:
Thank you, I will make these changes!